### PR TITLE
Update README.md: Updated the documentation link

### PR DESCRIPTION
**Pull request by 0xScratch**
New link seems to be this one -> https://panoptic.xyz/docs/intro

Also, there's a heading inside the README.md which says 'Force Exercisors'..Will it really be Exercisors? Or need to change it to 'Exercises'

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ The Panoptic protocol is noncustodial, has no counterparty risk, offers instanta
 - [Panoptic's Website](https://www.panoptic.xyz)
 - [Whitepaper](https://paper.panoptic.xyz/)
 - [Litepaper](https://intro.panoptic.xyz)
-- [Documentation](https://www.docs.panoptic.xyz)
+- [Documentation](https://panoptic.xyz/docs/intro)
```
Comment (jesperkristensen58) [inline README.md]: Let's use:

https://docs.panoptic.xyz

**Review by jesperkristensen58 — CHANGES_REQUESTED** [review]
one comment - otherwise looks good

**Comment by 0xScratch** [comment]
What about that 'Force Exercisors'?

**Comment by dyedm1** [comment]
> What about that 'Force Exercisors'? 

This is intended - just like liquidations and liquidators, exercises are actions while exercisors are actors. We use the same terminology internally and in our documentation.

LGTM on the other changes!

**Comment by 0xScratch** [comment]
Got it..Thanks!

**Pull request merged**